cli/clibase/cmd.go

```diff
@@ -47,6 +48,10 @@ type Cmd struct {
 	HelpHandler HandlerFunc
 }
 
+func (c *Cmd) Runnable() bool {
```

Comment:
This is never called

Reply:
It is in the clidocsgen

Reply:
I don't think this is needed, and a bit misleading since shouldn't we check if the command has a `Use` instead?

Right now it basically only removes usage from `coder provisionerd`, which doesn't make sense.

```diff
+## Usage
+
+```console
+provisionerd
+```
+
```